routes/api/budget.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+const express = require("express");
+const HttpStatus = require("http-status-codes");
+
+const interceptors = require("../interceptors");
+const models = require("../../models");
+
+const router = express.Router();
+
+router.get("/", async function (req, res) {
+  const row = await models.Budget.findAll();
+  res.json(row);
+});
+
+router.post("/", interceptors.requireLogin, async function (req, res) {
+  const user = req.user;
```

Comment:
When the interceptors.requireLogin function is used above, the user object for the logged in user will be accessible on the request object.